### DISCUSSION TITLE
Huge thanks for the nodelock feature !

### DISCUSSION THREAD

**Discussion by JbCourtois** [b-inary/postflop-solver#20] [General]
Node-locking is a key part of most poker solvers ; looking forward to use it on WASM Postflop !

Meanwhile, I'll try this on the engine, probably update the example file with a use case.
Great work !

**Comment by b-inary** [comment]
Thanks for watching the repository! I added [a brief documentation](https://b-inary.github.io/postflop_solver/postflop_solver/struct.PostFlopGame.html#method.lock_current_strategy) of related methods. [The test codes](https://github.com/b-inary/postflop-solver/blob/1d8386dfc04838f017ac8649d4803626e2595160/src/game.rs#L3556) may also be helpful.

I'll do my best to add this feature to WASM/Desktop Postflop as soon as possible!

**Comment by bkushigian** [comment]
Woah, I've been on endless paper deadlines and missed that nodelocking was implemented...this is huge, thanks @b-inary!
